### DISCUSSION TITLE
Support `multiple` in the AttachAction

### DISCUSSION THREAD

**Discussion by W1-PopelierE** [filamentphp/filament#11894] [Ideas]
I attached a `permission` relation to my `role` model. After playing around I noticed the `->multiple()` option is missing in the AttachAction.

```PHP
AttachAction::make()
    ->multiple() // missing

```
In my own project I have extended the AttachAction to fix this.
@danharrin , it may not be a major change, but as a newcomer, I would love to handle the development myself. :)

**Comment by danharrin** [comment]
Sounds good, please do it for AssociateAction too though
